### ISSUE TITLE
Use snake_case in json body

### ISSUE THREAD

**Issue by kektusini** [lolouk44/xiaomi_mi_scale#24]
Now in mqtt messages are sent in a very strange json format, where all the fields are like raw data:

```json
Publishing data to topic miScale/kektusini/weight: {"Weight":"83.65","BMI":"24.44","Basal Metabolism":"1733.59","Visceral Fat":"13.63","Lean Body Mass":"64.87","Body Fat":"23.41","Water":"52.54","Bone Mass":"3.27","Muscle Mass":"60.80","Protein":"20.15","Body Type":"Thick-set","Metabolic Age":"32","TimeStamp":"2020-07-03-15:54:09"}
```

**Describe the solution you'd like**
It would be better to use the snake_case format to describe json fields:

```json
{
   "weight":"83.65",
   "bmi":"24.44",
   "basal_metabolism":"1733.59",
   "visceral_fat":"13.63",
   "lean_body_mass":"64.87",
   "body_fat":"23.41",
   "water":"52.54",
   "bone_mass":"3.27",
   "muscle_mass":"60.80",
   "protein":"20.15",
   "body_type":"thick-set",
   "metabolic_age":"32",
   "timestamp":"2020-07-03-15:54:09"
}
```

**Comment by lolouk44** [comment]
Thanks for your feedback.
I don't understand the issue here, as one would _use_ the data from the MQTT topic and not _read_ it, unless it's for troubleshooting.
Even then, the real troubleshooting issue would be to check whether the message is being sent or not...
Or am I missing something?

**Comment by lolouk44** [comment]
Closing due to no reply

**Comment by vassilis-panos** [comment]
I think we need to reopen this. All attributes in home assistant are snake_case.

**Comment by lolouk44** [comment]
> All attributes in home assistant are snake_case

Do you mean "All attributes in home assistant **must** be in snake_case format"?
I've not seen any dev doc mentioning this...
That said I don't mind changing but that will create a breaking change...

**Comment by vassilis-panos** [comment]
![εικόνα](https://user-images.githubusercontent.com/4130346/92379656-8157b100-f110-11ea-8cb4-7285592d2a05.png)

**Comment by lolouk44** [comment]
Thanks I'll amend on my next update...

**Comment by lolouk44** [comment]
fixed in 0.1.8
